src/targets/gpu/hip_gemm_impl.cpp

```diff
@@ -633,9 +632,19 @@ int32_t hip_gemm_finalize(context& ctx,
                           float beta,
                           int32_t solution_idx)
 {
-    auto gemm_item   = hip_gemm_impl(output_shape, input_shapes, alpha, beta);
-    int32_t solution = gemm_item.tune(ctx, input_shapes);
-    hip_gemm_save_solution(ctx, output_shape, input_shapes, solution_idx);
+    int solution;
```

Comment:
Remove this variable and use `solution_idx` variable instead.

Reply:
Done.